modules/file-system/pre-existing-network-storage/outputs.tf

```diff
@@ -77,11 +81,21 @@ locals {
     "args"        = "\"not-used\" \"${local.gcsbucket}\" \"${var.local_mount}\" \"${var.fs_type}\" \"${var.mount_options}\""
     "content"     = file("${path.module}/scripts/mount.sh")
   }
+
+  mount_runner_daos = {
+    "type"        = "shell"
+    "content"     = file("${path.module}/scripts/mount-daos.sh")
+    "args"        = format("--local_mount=%s --mount_options=%s", var.local_mount, var.mount_options)
+    "destination" = "mount.sh"
```

Comment:
`mount.sh` is too generic, high changes for collision. Take a look at other mount scripts names in this file.

Reply:
Updated.